homeassistant/components/switchbot_cloud/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "switchbot_cloud",
+  "name": "SwitchBot Cloud",
+  "codeowners": ["@SeraphicRav"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/switchbot_cloud",
+  "iot_class": "cloud_polling",
+  "loggers": ["switchbot_cloud"],
```

Comment:
Is this logger correct?

Reply:
I see, I misunderstood the usage of this field

Reply:
I am not sure about how to use this actually

Reply:
https://developers.home-assistant.io/docs/creating_integration_manifest#loggers probably just the name of your lib

Reply:
Ok, even reading the doc, I did not understand more. I added the name of the lib. Thanks !